README.md

```diff
@@ -27,10 +27,11 @@ In your `Cargo.toml`, put:
 pathfinding = "0.1"
 ```
 
-Or if you need the edmondskarp algorithm you'll need to specify this by adding a specific feature:
+Or if you need the edmondskarp algorithm you'll need to specify this by adding the corresponding feature:
```

Comment:
The algorithm is named "Edmonds-Karp". I would suggest

    the Edmonds-Karp algorithm (edmondskarp)